source/compliance.txt

```diff
@@ -6,48 +6,189 @@ Compliance
+  You can set up {+qe+} using the following mechanisms:
+
+  - :manual:`Automatic Encryption <csfle-tutorial-automatic-encryption>`
+    enables you to perform encrypted read and write operations without
+    having to add explicit calls to encrypt and decrypt fields.
+  - :manual:`Manual Encryption <qe-tutorials-manual-encryption>` enables
+    you to perform encrypted read and write operations through your MongoDB
+    driver's encryption library. You must specify the logic for encryption
+    with this library throughout your application.
+
+.. _arch-center-compliance-data-sovereignty:
+
+Data Sovereignty
+~~~~~~~~~~~~~~~~
+
+{+service+} simplifies data sovereignty compliance through the following features:
+
+- Multi-cloud and multi-region {+clusters+}.
+
+  Use multi-region and multi-cloud {+clusters+} to distribute data across
+  geographically-diverse regions and cloud providers. To learn more,
+  see :atlas:`Create Multi-Cloud Multi-Region MongoDB Deployments for High
+  Availability and Workload Isolation <create-cluster-multi-region>`.
+
+  Create {+clusters+} in any of the 110+ cloud regions across the three
+  major cloud providers, |gcp|, |aws|, and |azure|. With {+service+},
+  you can deploy dedicated MongoDB {+clusters+} in a specific cloud
+  provider and region of your choice. This approach increases security,
+  reduces the risk of localized failures, and ensures data redundancy.
+
+  For example, a European company handling sensitive customer data can
+  choose to deploy their MongoDB {+cluster+} in a data center located
+  within the European Union, such as in Frankfurt, Germany, using |aws|
+  or another cloud provider. By doing so, the company ensures that all
+  data remains within EU borders and complies with the GDPR.
+
+- Global {+clusters+} with zoned sharding.
+
+  Use global {+clusters+} to control data storage locations. You can
+  partition and store data in your database into distinct zones, each
+  situated in a different geographical location. To learn more,
+  see :atlas:`Global Clusters <global-clusters>`.
+  
+  For example, you can store European customer data in Europe, while
+  storing U.S. customer data in the U.S. This allows you to comply with
+  data sovereignty regulations and reduces latency for users accessing
+  data from their respective regions.
+
+.. _arch-center-compliance-backup-distribution:
+
+Backup Snapshot Distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Multi-region snapshot distribution. Distribute backup snapshots and oplog data
+across multiple regions. For example, you can satisfy compliance requirements
+and store backups in different, air-gapped geographical locations to
+ensure disaster recovery in case of regional outages. To learn more,
+see :atlas:`Snapshot Distribution <snapshot-distribution>`.
+
+.. _arch-center-compliance-backup-compliance-policy:
+
+Backup Compliance Policy
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use |bcp| in {+service+} to secure business-critical data. You can prevent
```

Comment:
Reframe this as "you can" because this page isn't making recommendations (compliance recs will be in a future version)
```suggestion
You can use |bcp| in {+service+} to secure business-critical data. You can prevent
```